components/spinner/__docs__/storybook-stories.jsx

```diff
@@ -7,12 +7,17 @@ import Default from '../__examples__/default';
 
 const getSpinner = (props) => <Spinner {...props} />;
 
-const inverseContainer = {
+const inverseContainerStyle = {
 	backgroundColor: '#4bca81',
```

Comment:
I'd recommend using `#16325c` instead of the sickly green color to be consistent with the button examples